Makefile

```diff
@@ -18,7 +18,7 @@ $(MAKEFILE):
 
 # we still need to do this for windows
 bblfsh-client:
-	cd vendor/gopkg.in/bblfsh/client-go.v2 && make dependencies
+	cd vendor/gopkg.in/bblfsh/client-go.v3 && make dependencies
```

Comment:
Why the change in client-go version? If I am not mistaken we are still using v2.

Reply:
Nope, we're using v3 in master